code/processes/gateway.q

```diff
@@ -535,6 +535,18 @@ addserversfromconnectiontable[.servers.CONNECTIONS]
 // Join active .gw.servers to .servers.SERVERS table
 activeservers:{lj[select from .gw.servers where active;`handle xcol `w xkey .servers.SERVERS]}
 
+/function to tell orchestrator to scale up
+scaleup:{[procname]
+handle:first exec w from .servers.SERVERS where proctype=`orchestrator;
```

Comment:
I think you could/should probably use `.servers.gethandlebytype` here

```suggestion
handle:.servers.gethandlebytype[`orchestrator;`any];
```

(this will mean that if for whatever reason connection to orchestrator has been lost, it will attempt to re-open it first)